### PR TITLE
live hosts

### PR DESCRIPTION
**Pull request by Aamir-raza-1**
This pack scans live hosts on a given network using nmap. It returns total number of live hosts and a list of IP's of the live hosts on a network. It takes one argument that is the network, which is mandatory
Usage:
st2 run get_live_hosts.get_live_hosts network=<1.1.1.1/1> # give the network(a subnet) for listing the live hosts

### CODE REVIEW

File: packs/get_live_hosts/actions/get_live_hosts.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import nmap
+from st2actions.runners.pythonrunner import Action
+
+
+
+class check_live_hosts(Action):
```
Comment (Kami) [inline packs/get_live_hosts/actions/get_live_hosts.py]: Minor style thing - please use camel case for class name (`CheckLiveHostsAction`).

**Comment by Kami** [comment]
Same here - this action also fits better in the existing linux pack.

File: packs/linux/actions/get_live_hosts.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+name: get_live_hosts
+runner_type: run-python
+description: List all hosts on a given Network
+enabled: true
+entry_point: get_live_hosts.py
+parameters:
+  network:
+    type: string
+    description: "Network to query live hosts "
```
Comment (Kami) [inline packs/linux/actions/get_live_hosts.yaml]: Would be more explicit to say `Subnet of a network to scan for live hosts (e.g. 192.168.0.0/24)`. Probably also better to call the parameter `subnet`.

Comment (Aamir-raza-1) [inline packs/linux/actions/get_live_hosts.yaml]: yeah, You are right, I have changed the name from network to subnet and remove the main entry point.

**Pull request closed without merging**